src/constants.js

```diff
@@ -17,6 +17,7 @@ export const characters = [
     { name: 'jerry', frames: [160, 164, 162, 168, 161, 166] },
     { name: 'jordan', frames: [170, 174, 172, 178, 171, 176] },
     { name: 'jessie', frames: [180, 184, 182, 188, 181, 186] },
+    { name: 'yash', frames: [100, 100, 100, 100, 100, 100] },
```

Comment:
Unfortunately, this doesn't work.
At the coordinates which you have provided, there is no character sprite.

You can add a entire new one by adding a new png file. Double check the setup here to get an idea how it works